### ISSUE TITLE
[Outreachy Task Submission] Add Hidden Button Value Text for Close button in Post Details Modal

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4906]
# Intro
This is an Accessibility issue I found with the 'Close' button in the `post-details` modal. The `Close` button, despite being an SVG, has no value text.

## Issue
The button for closing the Post details modal has no value text. ***This is a violation of WCAG Success Criterion 1.1.1 (Level A) - Non-text content***. [See](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

## Description.
***Success Criterion 1.1.1 (Level A) - Non-text content*** requires that all non-text content presented to users has a text alternative that serves the equivalent purpose, except for situations where the content is intended to be omitted by the author or is decorative in nature.

## Impact
Accessibility software that can either convert text to visuals (for deaf people), and braille (for blind people) will not work. People who cannot understand the meaning of images/icons would struggle with this. Additionally, there would be no way to search for visual content in a text-based manner. 

## Fix
Add a value text to the button. I will raise a PR for this, shortly.
